### DISCUSSION TITLE
Unable to verify the first certificate

### DISCUSSION THREAD

**Discussion by IgetPM25lots** [GetPublii/Publii#1547] [Help]
My server updated their FTP certificate and Publii refuses to connect when the Certificates option under Settings is set to: Require a valid certificate for connection.
Using WSFTP, the connection works with the same credentials.
I need help to update the 'first certificate' in Publii to resolve this issue.  Thanks.

**Comment by dziudek** [comment]
@IgetPM25lots - most probably the certificate used on your FTP is self-signed and then will return issue during certificate validation, because it is not confirmed by any trused 3rd party.
